nifi-commons/nifi-utils/src/main/java/org/apache/nifi/util/FormatUtils.java

```diff
@@ -242,11 +244,13 @@ public static Instant parseToInstant(DateTimeFormatter formatter, String text) {
             throw new IllegalArgumentException("Text cannot be null");
         }
 
-        TemporalAccessor parsed = formatter.parseBest(text, Instant::from, LocalDateTime::from, LocalDate::from, LocalTime::from);
+        TemporalAccessor parsed = formatter.parseBest(text, Instant::from, LocalDateTime::from, LocalDate::from, YearMonth::from, Year::from, LocalTime::from);
```

Comment:
Minor adjustment, but I recommend moving the `YearMonth` and `Year` queries to the end, matching the switch statement that follows.

Reply:
Thanks for pointing out that important detail, looks good in that case.